superduper/components/datatype.py

```diff
@@ -194,12 +251,10 @@ class DataType(Component):
     :param media_type: The media type.
     """
 
-    type_id: t.ClassVar[str] = 'datatype'
     encoder: t.Optional[t.Callable] = None  # not necessary if encodable is file
     decoder: t.Optional[t.Callable] = None
-    info: t.Optional[t.Dict] = None
-    shape: t.Optional[t.Sequence] = None
-    directory: t.Optional[str] = None
+    info: t.Optional[t.Dict] = None  # TODO deprecate
+    directory: t.Optional[str] = None  # TODO needed?
```

Comment:
We don't need it

Reply:
Yes we'll deprecate in a following commit/ PR.